actionview/lib/action_view/helpers/translation_helper.rb

```diff
@@ -83,6 +83,7 @@ def translate(key, options = {})
         end
       rescue I18n::MissingTranslationData => e
         if remaining_defaults.present?
+          options.delete(:raise) unless options[:raise] == raise_error
```

Comment:
Looks like we need some more tests: they're currently all passing even without the `unless` conditional.